packages/ag-charts-website/src/features/api-documentation/components/OptionsNavigation.tsx

```diff
@@ -186,7 +186,7 @@ function NavProperty({
                 <>
                     <NavGroup depth={depth + 1}>
                         {isInterface
-                            ? processMembers(interfaceRef, config).map((childMember) => (
+                            ? processMembers(interfaceRef, config, []).map((childMember) => (
```

Comment:
you'll need to map the path and pull out the current config from the default otherwise this won't work
<img width="382" alt="image" src="https://github.com/ag-grid/ag-charts/assets/1083065/3904e933-aa40-416c-8fd5-aa267fcd5293">